src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -277,13 +277,8 @@ static std::tuple<std::vector<NodePtr>, std::vector<size_t>> ExtractExecutableNo
     std::vector<NodePtr> executableGraphNodes;
     for (size_t i = 0; i < graphNodes.size(); i++) {
         const auto& graphNode = graphNodes[i];
-        if ((!graphNode->isConstant() && CPU_DEBUG_CAPS_ALWAYS_TRUE(graphNode->isExecutable())) || // non-constant executable or
+        if (!graphNode->isConstant() || // non-constant executable or
```

Comment:
```suggestion
        if ((!graphNode->isConstant() && graphNode->isExecutable()) || // non-constant executable or
```
Check for isExecutable probably shouldn't be removed

Reply:
Right, my bad.
Just for information, this logic will also be updated in scope of another PR with memory reuse refactoring.